### PR TITLE
First Try Adding TTN Generic Node Sensor Edition

### PR DESCRIPTION
**Pull request by hallard**

### CODE REVIEW

File: mbed_app_template.json [hunk 1]
```diff
@@ -77,6 +77,17 @@
             "stm32wl-lora-driver.debug_tx": "PA_10",
             "stm32wl-lora-driver.debug_rx": "PA_9",
             "stm32wl-lora-driver.debug_invert": 1
+        },
+        "RAK3172SIP": {
```
Comment (jeromecoutant) [inline mbed_app_template.json:81]: Not related to this PR ?

**Review by jeromecoutant — APPROVED** [review]
All checks OK!

File: mbed_app_template.json (hunk 1 above)
Comment (hallard) [reply mbed_app_template.json:81]: yeah sorry forgot this one last time so just add this time (but it's template no no real issue)

**Comment by hallard** [comment]
hi @jeromecoutant, any chance to get it merged?

**Pull request merged**